### DISCUSSION TITLE
应用商店已安装的容器备份占用大量空间，但在面板上看不到对应的备份

### DISCUSSION THREAD

**Discussion by bluesky335** [1Panel-dev/1Panel#6888] [Q&A]
### 联系方式

bluesky335@foxmail.com

### 1Panel 版本

v1.10.20-lts

### 问题描述

面板上只可见一个备份
<img width="914" alt="image" src="https://github.com/user-attachments/assets/cca00774-f2ff-41e0-8b61-9707d6ca6fe7">
实际文件夹中有好多个
<img width="1045" alt="image" src="https://github.com/user-attachments/assets/9af9b42f-769f-44e3-9388-1f21ced3849c">
而且其中几个文件夹大小都不小
<img width="1016" alt="image" src="https://github.com/user-attachments/assets/723a4b48-b666-4d9e-b81d-dbe95ecb2bdd">

### 重现步骤

没有重现，猜测是因为我配置的 dockerhub 镜像源失效，容器升级失败导致。

### 期待的正确结果

正确显示备份，希望能提供设置，仅保留最新一次的备份

**Comment by wanghe-fit2cloud** [comment]
1. 进入 `日志审计` - `操作日志` 页面，搜索相关应用，查看具体操作记录。  
2. 目录中没有 tar.gz 后缀的文件，是自行通过命令解压生成的吗？

**Comment by bluesky335** [comment]
> 1. 进入 `日志审计` - `操作日志` 页面，搜索相关应用，查看具体操作记录。
> 2. 目录中没有 tar.gz 后缀的文件，是自行通过命令解压生成的吗？

1. 抱歉，开始我以为占用空间的是日志，我为了清理空间，在发现这个之前做了清理日志的操作。
2. 我没有手动操作过这个目录的任何文件

**Comment by wanghe-fit2cloud** [comment]
> > 1. 进入 `日志审计` - `操作日志` 页面，搜索相关应用，查看具体操作记录。
> > 2. 目录中没有 tar.gz 后缀的文件，是自行通过命令解压生成的吗？
> 
> 1. 抱歉，开始我以为占用空间的是日志，我为了清理空间，在发现这个之前做了清理日志的操作。
> 2. 我没有手动操作过这个目录的任何文件

我在本地环境测试了几个应用的升级场景，均未复现上述问题。你那边还存在该文件吗？如果在的话，我们可以远程查看一下。

**Comment by bluesky335** [comment]
> > > 1. 进入 `日志审计` - `操作日志` 页面，搜索相关应用，查看具体操作记录。
> > > 2. 目录中没有 tar.gz 后缀的文件，是自行通过命令解压生成的吗？
> > 
> > 
> > 
> > 1. 抱歉，开始我以为占用空间的是日志，我为了清理空间，在发现这个之前做了清理日志的操作。
> > 2. 我没有手动操作过这个目录的任何文件
> 
> 我在本地环境测试了几个应用的升级场景，均未复现上述问题。你那边还存在该文件吗？如果在的话，我们可以远程查看一下。

抱歉，已经清理掉了😂

**Comment by wanghe-fit2cloud** [comment]
好的，如果该问题再次复现，在这里继续讨论。
先将该 issue 转为 Discusstion 了。
